web/pdf_find_bar.js

```diff
@@ -51,16 +51,21 @@ class PDFFindBar {
     this.findField.addEventListener("input", () => {
       this.dispatchEvent("");
     });
+    this.findField.addEventListener("keydown", e => {
+      e.stopPropagation();
+    });
```

Comment:
Did you successfully test this, since I'd expect that doing this will break the "keydown"-handler just below?
Those keyboard shortcuts have previously always worked also when *the input is focused*, which this patch now breaks.